backends/bmv2/bmv2stf.py

```diff
@@ -73,14 +71,16 @@ def ByteToHex(byteStr):
     return ''.join( [ "%02X " % ord( x ) for x in byteStr ] ).strip()
 
 def HexToByte(hexStr):
```

Comment:
I don't think this method is being used at all any more, since you replaced it with calls to `bytes.fromhex`.

Reply:
removed.